index.html

```diff
@@ -113,186 +101,404 @@
         system from entering a power-saving state.
       </p>
       <p>
-        This specification deals specifically with the screen wake lock which
-        prevents the screen(s) of the device from entering a power-saving state.
+        This specification defines two <dfn data-lt="wake lock type">wake lock
+        types</dfn>:
       </p>
+      <ol>
+        <li>
+          <dfn>Screen wake lock</dfn> which prevents device's screen from
+          turning off so that the user can still see the information displayed
+          on the screen.
+        </li>
+        <li>
+          <dfn>System wake lock</dfn> which prevents device's CPU from entering
+          a standby mode so that the programs can continue running.
+        </li>
+      </ol>
+      <div class="note">
+        One type of wake lock can imply the effects of another: for example,
+        screen wake lock logically implies that the program which displays
+        information on the screen continues running, as if the system wake lock
+        were also applied. But to avoid dependence on such implementation
+        details which may not always be true, this specification treats all
+        types of wake locks as independent.
+      </div>
+      <p>
+        A <a>user agent</a> SHOULD support <a>screen wake lock</a> and MAY
+        support <a>system wake lock</a>.
+      </p>
```

Comment:
I don't think we should express it that way; I would suggest dropping that sentence entirely.